src/lib/ui/thermostat/Temperature.tsx

```diff
@@ -0,0 +1,26 @@
+interface TemperatureProps {
+  fahrenheit: number
+  celsius: number
+  unit: 'fahrenheit' | 'celsius'
+  hideUnit?: boolean
+}
+
+export function Temperature({
+  fahrenheit,
+  celsius,
+  unit,
+  hideUnit,
+}: TemperatureProps): JSX.Element {
+  const temperature = unit === 'fahrenheit' ? fahrenheit : celsius
+  const degree = unit === 'fahrenheit' ? t.degreeFahrenheit : t.degreeCelsius
+  return (
+    <span>
+      {Math.trunc(temperature)}°{hideUnit != null && degree}
+    </span>
+  )
+}
+
+const t = {
+  degreeFahrenheit: 'F',
+  degreeCelsius: 'C',
+}
```

Comment:
```suggestion
interface TemperatureProps {
  fahrenheit: number
  celsius: number
  unit: 'fahrenheit' | 'celsius'
  disableUnit?: boolean
}

export function Temperature({
  fahrenheit,
  celsius,
  unit,
  disableUnit = false,
}: TemperatureProps): JSX.Element {
  const temperature = unit === 'fahrenheit' ? fahrenheit : celsius
  const degree = unit === 'fahrenheit' ? t.degreeFahrenheit : t.degreeCelsius
  return (
    <span>
      {Math.trunc(temperature)}{!disableUnit && degree}
    </span>
  )
}

const t = {
  degreeFahrenheit: '°F',
  degreeCelsius: '°C',
}
```

Reply:
This changes the underlying behavior by removing the degree symbol. I added this prop for the Thermostat card which shows the unit itself elsewhere, but still uses the degree symbol indicating its a temperature. Removing the degree symbol would just make it a normal number, essentially (unless the degree is being rendered in another context but I think that's out of scope for this component anyway)

Reply:
I'll change the prop name though 👍

Reply:
When would we ever display `4°` instead of `4°C` though?

Reply:
Sorry, I mistakenly thought the unit wasn't shown in the `TemperatureCard`—think there's a bug in that PR. I can remove this behavior altogether unless you think there's a good reason to keep it.

Reply:
Ok, sound good. Yea I agree if we a temp is shown without a unit that sounds like a bug.